core/shard/src/lib.rs

```diff
@@ -28,8 +32,7 @@ pub struct SignedShardBlockHeader {
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct ShardBlock {
     pub header: ShardBlockHeader,
-    pub transactions: Vec<SignedTransaction>,
-    pub receipts: Vec<ReceiptTransaction>,
+    pub transactions: Vec<Transaction>,
 }
```

Comment:
resulting receipts that we will need to send out to other shards are still needed.

Reply:
Fixed in 4bed468